quality-assurance/safety-assurance-guidance.md

```diff
@@ -1,44 +1,47 @@
+If teams need more information about the safety context of their work, they will request that it is provided by product owners.  
 
-See [Bowtie in four easy steps](https://www.youtube.com/watch?v=PHbLQWqojC8)
+# Identifying safety-related projects
 
-## How should we use it
+A team’s product owner is accountable for establishing whether their current delivery is safety-related.
 
-### At the start of a project
+If the delivery is safety-related, then the team’s “definition of done” must contain a statement to the effect that safety modelling must have been carried out for every PBI (work item)
 
-* Carry out a bowtie analysis of the project to identify potential areas of safety that need to be considered (there are a number of safety assurance experts who can help in carrying out bowtie analysis).
+# Process – Safety Modelling
 
-### During Sprints
+When developing safety-related software, the following process must be followed.
 
-* Tag relevant PBI's with a `safety` tag along with details of how the safety risk can be mitigated. If it is not clear whether there is a safety impact use a `safety pending` tag until this ambiguity is resolved.
-* Consider testing the risk mitigation. There will be a balance between effort required to test against the likelihood and impact.
-* Let the PO make final decisions on the safety impact/consequence. The PO will engage with other SME's to determine if the safety risk is acceptable. This may  
-  mean that although there is a potential safety risk that risk is deemed manageable or may require remedial action. This detail should be captured in a lightweight 
-  manner within the relevant PBI.
- 
-## General Principles 
+1. If a PBI is considered as potentially relevant to safety, it must be marked in the work tracking system with the standard tag ‘Safety Pending’.
+2. The development team must then discuss safety requirements for that PBI.  This discussion may be at any team event (e.g. refinement, threat-modelling, or a specific safety meeting).
+3. Under the direction of the test engineer, the team must discuss whether a failure condition relating to the PBI can cause an error or delay in the product.  If it is decided that this is the case, the PBI must be marked with the standard tag ‘Safety’.  If it is not the case, then all safety tags may be removed.
+4. The team treats the potential hazard as a mini bowtie analysis, considering i) Barriers to the failure condition happening ii)	Mitigations to prevent the failure condition creating error or delay
```

Comment:
As the bowtie stuff has been removed, does this need further clarification or linking?

Reply:
You can achieve these bullets better with markdown (indenting and * or -)

Reply:
re:  bowtie, yes I need to write some content for this; in the meantime I've just put a link to a youtube explanation